py3status/modules/netdata.py

```diff
@@ -3,32 +3,38 @@
 Display network speed and bandwidth usage.
 
 Configuration parameters:
-    cache_timeout: how often we refresh this module in seconds (default 2)
-    low_speed: threshold (default 30)
-    low_traffic: threshold (default 400)
-    med_speed: threshold (default 60)
-    med_traffic: threshold (default 700)
-    nic: the network interface to monitor (default None)
-
-Color options:
-    color_bad: Rate is below low threshold
-    color_degraded: Rate is below med threshold
-    color_good: Rate is med threshold or higher
+    cache_timeout: refresh interval for this module (default 2)
+    format: display format for this module
+        *(default '[\?color=down LAN(Kb): {down}↓ {up}↑] '
+        '[\?color=total T(Mb): {download}↓ {upload}↑ {total}↕]'*
+    nic: network interface to use (default None)
+    thresholds: color thresholds to use
+        *(default '{down': [(0, "bad"), (30, "degraded"), (60, "good")], '
```

Comment:
`'` int the wrong place `{'down':` the whole thing is a dict not a string - you have extra `'`s knocking about